documentation/setup/Dune.md

```diff
@@ -66,8 +71,14 @@ flag, e.g
 ```
 
 There is a very nice command `dune coq top` that will call `coqtop` with the
-correct flags, but unfortunately Proof General currently does not support it
-(see issue [477](https://github.com/ProofGeneral/PG/issues/477)). We hope this
-issue gets resolved soon as this would remove any need to configure PG/coqtop
-beyond telling it to call `dune coq top`.
+correct flags.
+
+### Proof General
+Unfortunately, Proof General currently does not support it (see issue [477](https://github.com/ProofGeneral/PG/issues/477)).
+We hope this issue gets resolved soon as this would remove any need to configure PG/coqtop beyond telling it to call `dune coq top`.
 
+## Generating documentation
+To generate documentation with Dune for UniMath (or another theory like SetHITs or TypeTheory), perform the following steps:
+1. Navigate to the [`UniMath`](../../UniMath/) directory *inside* the repository. This is the root directory of the code: the directory with the [`dune`](../../UniMath/dune) file that contains the `coq.theory` stanza. For other theories, this is for example [`SetHITs/code`](../../SetHITs/code) or [`TypeTheory/TypeTheory`](../../TypeTheory/TypeTheory/).
+2. Execute the command `$ dune build UniMath.html` (adjust accordingly for your theory of choice).
```

Comment:
How does this adjustment work? Can we give a second example to clarify this?

Reply:
Sure. `$ dune build TypeTheory.html` or `$ dune build SetHITs.html`. What is the confusion?